src/main/kotlin/org/neo4j/graphql/Predicates.kt

```diff
@@ -88,98 +89,99 @@ fun arrows(outgoing: Boolean?): Pair<String, String> {
 }
 
 
-data class RelationshipInfo(val objectType:GraphQLObjectType, val directive:Directive, val type:String, val out:Boolean?, val startField: String? = null,val endField: String? = null, val isRelFromType: Boolean = false) {
+data class RelationshipInfo(val objectType: GraphQLObjectType, val directive: Directive, val type: String, val out: Boolean?, val startField: String? = null, val endField: String? = null, val isRelFromType: Boolean = false) {
     val arrows = arrows(out)
-    val label = objectType.name
+    val label: String = objectType.name
```

Comment:
don't need the type info

Reply:
IntelliJ annotes this with: Declaration has type inferred from a platform call, which can lead to unchecked nullability issues. Specify type explicitly as nullable or non-nullable.